server/blob/blob-s3/src/main/java/org/apache/james/blob/objectstorage/aws/S3BlobStoreDAO.java

```diff
@@ -411,7 +409,7 @@ public Mono<Void> deleteAllBuckets() {
         return Mono.fromFuture(client::listBuckets)
             .publishOn(Schedulers.parallel())
             .flatMapIterable(ListBucketsResponse::buckets)
-                .flatMap(bucket -> deleteResolvedBucket(BucketName.of(bucket.name())), DEFAULT_CONCURRENCY)
+                .flatMap(bucket -> deleteResolvedBucket(ResolvedBucketName.of(bucket.name())), DEFAULT_CONCURRENCY)
```

Comment:
**❔ question:** ‏ I realize this was the case before this MR but is it normal that we don't go through 
`ResolvedBucketName resolvedBucketName = bucketNameResolver.resolve(bucketName);` here whil we do for all other public calls ? I don't necessarily expect this to be fixed in this MR, just trying to understand if there is something that needs fixing at all :D

Reply:
We are listing buckets with S3 API then we call a delete for each bucket.
In S3, there are only buckets with "resolved names".